### PR TITLE
Rewrite the GraphQL service scenario in tooling approach

### PR DESCRIPTION
**Pull request by praneesha**
## Purpose
Rewrite the GraphQL service scenario in tooling approach.
> Fixes #

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: swan-lake/resources/featured-scenarios/write-a-graphql-api-with-ballerina.md [hunk 1]
```diff
@@ -8,177 +8,184 @@ active: write-a-graphql-api-with-ballerina
...
+    ```
+    $ bal new write-a-graphql-api
+    ```
 
-Before writing the GraphQL service, let's create a data source for the project. This will mimic a database that stores the data for the service using an in-memory table in Ballerina as the data source.
+    You should see the output similar to the following.
 
-### Define the types for the data source
+    ```
+    package name is derived as 'write_a_graphql_api'. Edit the Ballerina.toml to change it.
```
Comment (DimuthuMadushan) [inline swan-lake/resources/featured-scenarios/write-a-graphql-api-with-ballerina.md:51]:
```suggestion
    package name is derived as 'write_a_graphql_api'. Edit the Ballerina.toml to change it.
```
Shouldn't this be outside the code block

File: swan-lake/resources/featured-scenarios/write-a-graphql-api-with-ballerina.md [hunk 2]
```diff
@@ -8,177 +8,184 @@ active: write-a-graphql-api-with-ballerina
...
-```
-
-In this code:
-- The record type `CovidEntry` is defined.
-- A single entry will include the ISO code for the location (`isoCode`), name of the country (`country`), number of cases (`cases`), number of deaths (`deaths`), number of recovered cases (`recovered`), and the number of active cases (`active`).
+    ```
+    .
+    │   ├── Ballerina.toml
+    │   └── main.bal
+    ```
```
Comment (DimuthuMadushan) [inline swan-lake/resources/featured-scenarios/write-a-graphql-api-with-ballerina.md:62]:
```suggestion
    ```
    .
    │   ├── Ballerina.toml
    │   └── main.bal
    ```
```
Shall we add the package name as well.

File: swan-lake/resources/featured-scenarios/write-a-graphql-api-with-ballerina.md [hunk 3]
```diff
@@ -8,177 +8,184 @@ active: write-a-graphql-api-with-ballerina
...
     }
+    ```
 
-    resource function get deaths() returns decimal? {
-        if self.entryRecord.deaths is decimal {
-            return self.entryRecord.deaths / 1000;
-        }
-        return;
-    }
+The generated record and the table will be as follows.
```
Comment (DimuthuMadushan) [inline swan-lake/resources/featured-scenarios/write-a-graphql-api-with-ballerina.md:151]:
```suggestion
    The generated record and the table will be as follows.
```
check the indentation in following section.

Comment (praneesha) [reply swan-lake/resources/featured-scenarios/write-a-graphql-api-with-ballerina.md:151]: This is not indented as it belongs to both the 1st and 2nd bullet points.

File: swan-lake/resources/featured-scenarios/write-a-graphql-api-with-ballerina.md (hunk 1 above)
Comment (praneesha) [reply swan-lake/resources/featured-scenarios/write-a-graphql-api-with-ballerina.md:51]: This is the output generated on the CLI.

File: swan-lake/resources/featured-scenarios/write-a-graphql-api-with-ballerina.md (hunk 2 above)
Comment (praneesha) [reply swan-lake/resources/featured-scenarios/write-a-graphql-api-with-ballerina.md:62]: Will update after changing the package names as per https://github.com/ballerina-platform/ballerina-dev-website/pull/8664#discussion_r1496724115.

File: swan-lake/resources/featured-scenarios/write-a-graphql-api-with-ballerina.md [hunk 4]
```diff
@@ -8,177 +8,184 @@ active: write-a-graphql-api-with-ballerina
...
-├── covid19
-│   ├── tests
-│   │   └── service_test.bal
-│   ├── Ballerina.toml
-│   └── service.bal
-```
-
-## Create the data source 
+    ```
+    $ bal new write-a-graphql-api
```
[the remark this answers is not in the document]
Comment (praneesha) [reply swan-lake/resources/featured-scenarios/write-a-graphql-api-with-ballerina.md:45]: Had discussions and updated to `graphql-service`.

**Review by ThisaruGuruge — APPROVED** [review]
LGTM, with a minor comment.

File: swan-lake/resources/featured-scenarios/write-a-graphql-api-with-ballerina.md [hunk 5]
```diff
@@ -34,151 +32,141 @@ The `Query` type has two fields.
...
 
 ```ballerina
 import ballerina/graphql;
 
 service /covid19 on new graphql:Listener(9000) {
-
 }
 ```
 
 >**Info:** The path of this service is defined as `/covid19`. If you want to host the service on the
```
[the remark this answers is not in the document]
Comment (praneesha) [reply swan-lake/resources/featured-scenarios/write-a-graphql-api-with-ballerina.md:168]: Removed.

**Pull request merged**